### PR TITLE
Adds the size of the data to the extension logger

### PR DESCRIPTION
**Pull request by orta**
Adds the ability you log out the size of a request received by MP. Does not do it in cached requests, because there aren't headers for the size.

Uncached:
screen shot 2019-02-08 at 1 17 09 pm

Cached:
screen shot 2019-02-08 at 1 17 12 pm

### CODE REVIEW

File: src/lib/loaders/api/extensionsLogger.ts [hunk 1]
```diff
@@ -68,3 +68,10 @@ export const fetchLoggerRequestDone = requestID => () => {
 }
 
 export default extensionsLogger
+
+export const formatBytes = (bytes: number) => {
+  if (bytes < 1024) return bytes + " B"
+  else if (bytes < 1048576) return (bytes / 1024).toFixed(3) + " KB"
+  else if (bytes < 1073741824) return (bytes / 1048576).toFixed(3) + " MB"
+  else return (bytes / 1073741824).toFixed(3) + " GB"
```
Comment (mzikherman) [inline src/lib/loaders/api/extensionsLogger.ts]: maybe this GB line should be "please dont"

**Comment by mzikherman** [comment]
Looks great! Possibly a real ❌ (missing stub?)

File: src/lib/loaders/api/extensionsLogger.ts (hunk 1 above)
Comment (orta) [reply src/lib/loaders/api/extensionsLogger.ts]: `GB!!!!!`

**Comment by orta** [comment]
OK, this is back and will be greening 👍

File: src/lib/loaders/api/extensionsLogger.ts (hunk 1 above)
Comment (mzikherman) [reply src/lib/loaders/api/extensionsLogger.ts]: "if you are reading this, we have a problem. Gravity is already down"

**Pull request merged**